here-naksha-app-service/src/test/java/com/here/naksha/app/service/NakshaAppTest.java

```diff
@@ -919,6 +925,89 @@ void tc0816_testGetByTileWithUnsupportedTileType() throws Exception {
     readFeaturesByTileTestHelper.tc0816_testGetByTileWithUnsupportedTileType();
   }
 
+  @Test
+  @Order(16)
+  void tc0900_testSearchFeatures() throws URISyntaxException, InterruptedException, JSONException, IOException {
+    // Test API : GET /hub/spaces/{spaceId}/tile/{type}/{tileId}
+    // Validate features getting returned for given Tile and given single tag value
+    String streamId;
+    HttpResponse<String> response;
+
+    // Given: Storage (mock implementation) configured in Admin storage
+    final String storageJson = loadFileOrFail("ReadFeatures/Search/TC0900_search/create_storage.json");
+    streamId = UUID.randomUUID().toString();
+    response = nakshaClient.post("hub/storages", storageJson, streamId);
+    assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Storage");
+
+    // And: EventHandler (uses above Storage) configured in Admin storage
+    final String handlerJson = loadFileOrFail("ReadFeatures/Search/TC0900_search/create_event_handler.json");
+    streamId = UUID.randomUUID().toString();
+    response = nakshaClient.post("hub/handlers", handlerJson, streamId);
+    assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Event Handler");
+
+    // And: Space (uses above EventHandler) configured in Admin storage
+    final String spaceJson = loadFileOrFail("ReadFeatures/Search/TC0900_search/create_space.json");
+    final Space space = TestUtil.parseJson(spaceJson, Space.class);
+    streamId = UUID.randomUUID().toString();
+    response = nakshaClient.post("hub/spaces", spaceJson, streamId);
+    assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Space");
+
+    // And: New Features persisted in above Space
+    String bodyJson = loadFileOrFail("ReadFeatures/Search/TC0900_search/create_features.json");
+    streamId = UUID.randomUUID().toString();
+    response = nakshaClient.post("hub/spaces/" + space.getId() + "/features", bodyJson, streamId);
+    assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating new Features");
+
+    // And: search query
+    final String tagsQueryParam = "tags=one+four";
+    final String expectedBodyPart = loadFileOrFail("ReadFeatures/Search/TC0900_search/search_response.json");
+    streamId = UUID.randomUUID().toString();
+
+    // When: Get Features By Tile request is submitted to NakshaHub
+    response = nakshaClient.get("hub/spaces/" + space.getId() + "/search" + "?" + tagsQueryParam, streamId);
+
+    // Then: Perform assertions
+    Assertions.assertEquals(200, response.statusCode());
```

Comment:
let's use standardAssertions() function the way we are using in other TestHelper class, so we have uniform assertions at all places

Reply:
done